src/test/java/com/datatheorem/mobileappsecurity/jenkins/plugin/SendBuildToDataTheoremPublisherTest.java

```diff
@@ -19,18 +19,6 @@
 public class SendBuildToDataTheoremPublisherTest {
 
     private final String buildName = "t*est-*.apk";
-    private final String sourceMapToUpload = "t*est-*.txt";
```

Comment:
Do we actually want to remove that from the tests?

Reply:
no we don't